Frontend/ui-customer/src/Flow.purs

```diff
@@ -490,10 +490,19 @@ currentFlowStatus = do
                        , sourceAddress = flowStatusData.sourceAddress
                        , destinationAddress = flowStatusData.destinationAddress }
                 })
-            Nothing -> do
-              updateFlowStatus SEARCH_CANCELLED
-        else do
-          updateFlowStatus SEARCH_CANCELLED
+            Nothing -> cancelFindingQuotes
+        else cancelFindingQuotes
+
+    cancelFindingQuotes :: FlowBT String Unit
+    cancelFindingQuotes = do
+      (GlobalState globalState) <- getState
+      let homeScreenState = globalState.homeScreen
+          updatedHomeScreenState = if homeScreenState.props.customerTip.enableTips then 
```

Comment:
move if inside like this `isPopUp = homeScreenState.props.customerTip.enableTips then TipsPopUp else ConfirmBack`

Reply:
here we are not only updating `isPopUp` but using `tipEnabledState` as well based on condition